### ISSUE TITLE
add Recently Played section.

### ISSUE THREAD

**Issue by Root1-2** [digimezzo/dopamine#671]
Just like in the Windows Groove (Music) Player. It will be very helpful feature to play the song that I usually listened.

**Comment by digimezzo** [comment]
@Root1-2 I love the idea. Thank you! I'm adding this to the pipeline. I'll frist need to reorganize the UI a bit, because the menu has many items already. So this will not be added in the next 2 previews, but most likely around preview 35.

**Comment by Root1-2** [comment]
Yeah sure thing man. Take your time. Another thing, I have imported over 14000 songs in dopamine. Now the whole software has become way much slower and less responsive. Is there anything I can do about it?

**Comment by digimezzo** [comment]
@Root1-2 I wasn't expecting that. I have created a separate issue #673 just now to be able to track it individually.
